training/distributed_training/pytorch/model_parallel/gpt2/smp-train-gpt-simple-sharded-data-parallel.ipynb

```diff
@@ -0,0 +1,768 @@
+{
```

Comment:
<p>Use present tense (i.e. you learn, not you'll learn), is proposed instead of was proposed, etc. Please scan through the entire notebook and change everything to present tense</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3622/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>